shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -28,6 +24,10 @@
 public class ShadowBiometricManager {
 
   protected boolean biometricServiceConnected = true;
+  private Integer enrolledBiometrics;
```

Comment:
Why `Integer` and `Boolean` instead of `int` and `bool`?

Reply:
It is because i need to check the value is set or null in order to return different error results

Reply:
Can we set false as default value for those fields? @hoisie What do you think?